docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge
+services:
+  pythonapp:
+    image: app:1.2
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8080:8080"
+    environment:
+      - APP_ENV=Development
+    networks:
+      - db-data-net
+    depends_on:
+      - mysql
+    restart: unless-stopped
+  mysql:
+    image: my-sql:latest
+    build: 
+      context: .
+      dockerfile: Dockerfile.mysql
+    container_name: my-sql
+    ports:
+      - "3306:3306"
+    volumes:
+      - db-data:/var/lib/mysql
+    environment:
+      - MYSQL_ROOT_PASSWORD=1234
+      - MYSQL_DATABASE=app_db
+      - MYSQL_USER=app_user
+      - MYSQL_PASSWORD=1234
+    networks:
+      - db-data-net
+volumes:
+  db-data:
```

Comment:
![image](https://github.com/user-attachments/assets/8c53c020-a5c9-4c53-90fb-8552defb7953)
Add Eof(End-of-file), need to add a blank line at the end